data/schema.graphql

```diff
@@ -1264,6 +1287,15 @@ enum BuyerOutcomeTypes {
 }
 
 type BuyersPremium {
+  # A globally unique ID.
+  __id: ID!
+
+  # A type-specific ID.
+  id: String!
+
+  # A type-specific Gravity Mongo Document ID.
+  _id: String!
+
```

Comment:
lols yeah, we have 3 types of IDs, let's think of it in terms of an Artist:

* `id` - `banksy` the usual slug-likeID
* `_id` - The mongo ID ( every JSON object in Mongo has a UUID)
* `__id` - I believe in _some_ models we had already used `_id` to _not_ mean the mongoID and so we had to go a step further and have a _really_ UUID (nearly always the mongoID, but if we have multiple DBs then in theory we can have clashes)